jib-core/src/test/java/com/google/cloud/tools/jib/registry/credentials/DockerConfigCredentialRetrieverTest.java

```diff
@@ -47,6 +47,8 @@ public void setUp()
           CredentialHelperNotFoundException, IOException {
     dockerConfigFile = Paths.get(Resources.getResource("json/dockerconfig.json").toURI());
 
+    Mockito.when(mockDockerConfig.getCredentialHelperFor(Mockito.any()))
```

Comment:
I think this should have specific arguments passed instead of `any` like the tests had before with `Mockito.when`.